torch/csrc/distributed/c10d/ProcessGroupNCCL.cpp

```diff
@@ -647,7 +647,7 @@ ProcessGroupNCCL::ProcessGroupNCCL(
   desyncDebug_ = parseEnvVarFlag(NCCL_DESYNC_DEBUG) ||
       (dist_debug_level_ >= DebugLevel::Detail);
 #ifdef ENABLE_NCCL_ERROR_CHECKING
-  enableTiming_.store(parseEnvVarFlag(NCCL_ENABLE_TIMING) || desyncDebug_);
```

Comment:
Well. I think this PR changed the logic somehow, https://github.com/pytorch/pytorch/pull/107233/files#diff-c4e79e1dee3eea72b89d4471b91de8a7dae284e27bd5c32ed3e5296a561e4ee9L1563. So if you remove this, desyncDebug_ will not log event start and end...

Reply:
ah good point. Let me look into it

Reply:
@fduwjj I think it'll skip logging the start of operation but it should still keep the end report?

Reply:
I see. But I guess you do need start event?